pyprof/prof/misc.py

```diff
@@ -129,7 +129,7 @@ def __init__(self, d):
         self.op_ = op
         self.args = args
 
-        assert (mod == "Tensor")
+        assert (mod == "Tensor" or mod == 'torch'), "Mod {} marker {}".format(mod, marker)
```

Comment:
Separate PR.

Reply:
Why?

Reply:
@dzier suggestion of using 1 PR for one isolated issue/feature.

Reply:
Commit really, you have multiple commits per PR, just need to make sure we don't squash the commits.